test/command/suite/normalizers/nfkc/unify_alphabet_diacritical_mark/a/latin_extended_additional.test

```diff
@@ -0,0 +1,6 @@
+#@add-substitution /NormalizerNFKC(\d*)/ "NormalizerNFKC#{ENV['NFKC'] || '100'}" "NormalizerNFKC"
+normalize \
+  'NormalizerNFKC("unify_alphabet_diacritical_mark", true)' \
+  "ḀḁẠạẢảẤấẦầẨẩẪẫẬậẮắẰằẲẳẴẵẶặÅ" \
```

Comment:
Is `Å` duplicated? 

```suggestion
  "ḀḁẠạẢảẤấẦầẨẩẪẫẬậẮắẰằẲẳẴẵẶặ" \
```

Reply:
fix: [7fd5baa](https://github.com/groonga/groonga/pull/1859/commits/7fd5baa7c855398c527e92afcca63b64731babf4) Thanks!